components/apimgt/api-store-web/src/site/themes/fancy/templates/api/throttling/template.jag

```diff
@@ -120,10 +120,11 @@ var i, length, app, thumbURL, api = outputs.api, templatesList = outputs.uriTemp
 
 
                                 <% getAuthVal=putAuthVal=postAuthVal=deleteAuthVal=optionAuthVal=""; %>
+                                <% getThrottlingTier=postThrottlingTier=deleteThrottlingTier=optionsThrottlingTier=putThrottlingTier=""; %>
```

Comment:
I believe the idea here is to initialize multiple variables at once? Would be much readable and elegant if the following format was used.

String getThrottlingTier, postThrottlingTier, deleteThrottlingTier = "";